### PR TITLE
Final fix for check_include_file

### PR DESCRIPTION
**Pull request by cwebster2**
Per disucssion with arclance in #166, the call to check_include_file has
been migrated back to check_include_files and the required define set in
CMAKE_REQUIRED_DEFINITIONS.  This properly removes the dependancy on
the check_include_file module.  Third time is a charm!

### CODE REVIEW

File: cmake/ConkyPlatformChecks.cmake [hunk 1]
```diff
@@ -141,7 +141,8 @@ if(BUILD_MYSQL)
 endif(BUILD_MYSQL)
 
 if(BUILD_WLAN)
-	check_include_file(iwlib.h IWLIB_H -D_GNU_SOURCE)
+        set(CMAKE_REQUIRED_DEFINITIONS -D_GNU_SOURCE)
```
Comment (brndnmtthws) [inline cmake/ConkyPlatformChecks.cmake]: Indentation is off here.

**Comment by cwebster2** [comment]
soft tab switched to hard tab, sorry about that.  force pushed to update the PR commit.

**Comment by cwebster2** [comment]
@brndnmtthws spacing fixed, what a headache :)

**Comment by brndnmtthws** [comment]
Cool, looks good. I'll wait to hear back from @arclance.

**Comment by arclance** [comment]
It configured, built, and ran without problems with this change for me.

**Comment by brndnmtthws** [comment]
Thanks!

**Pull request merged**